### ISSUE TITLE
Pokemon Colosseum bonus disc (JPN) hangs during loading

### ISSUE THREAD

**Issue by Pereb** [FIX94/Nintendont#195]
The game presents you with a menu with a bunch of promotional videos and the Celebi transfer program. Upon selecting the latter, the screen goes black and hangs. The videos work fine for me.

https://www.youtube.com/watch?v=bVyTw2Vj67o <- at the 0:25 mark is when it hangs

Not sure if I'm doing something wrong but I haven't found any fix nor setting to get it working.

**Comment by carnage702** [comment]
post all your nintendont settings and the number of the nintendont rev you are using.

**Comment by Pereb** [comment]
I'm using rev 3.383.

Cheats Off
Debugger Off
Debugger Wait Off
Memcard emulation On
Cheat path Off
Force widescreen Off
Force progressive Off
Auto boot Off
Unlock Read Speed Off
OSReport Off
WiiU Widescreen Off
Drive read LED Off
Log Off
MaxPads 4
Language Auto
Video Auto
Memcard Bloicks 251
Memcard Multi Off
Native control On
Video Width Auto
Screen position 0
Patch PAL50 Off

Turning off memory card emulation doesn't help either.

**Comment by FIX94** [comment]
it should be all good now, needed some special patching just like colosseum and xd themself, just update to the latest version and if everything works just close this issue.

**Comment by Pereb** [comment]
Thanks! Updated to 3.384. It no longer hangs when selecting the Celebi transfer program, however memory card emulation doesn't work (when attempting to load a Colosseum save, it throws an error saying there is no memory card in slot A). Tried 59 and 251 blocks, individual and multi. I don't have a real card to test it with unfortunately.

**Comment by FIX94** [comment]
maybe 385 works, colosseum also has a special memory card patch to even work and I just copied it over to the bonus disc.

**Comment by Pereb** [comment]
Works flawlessly. Thanks!

**Comment by SLIMEMASTER000** [comment]
im having this same issue and even on a real mem card that has been reformated to the JP gcn mem format its just a black screen at the celebi option... please help

**Comment by carnage702** [comment]
> im having this same issue and even on a real mem card that has been reformated to the JP gcn mem format its just a black screen at the celebi option... please help

just use an emulated memory card and pit the save there.

**Comment by CPDantesco** [comment]
Hi, I'm having the same problem. I have an original Colosseum Jap copy, I completed the game and Celebi's ingame requiriments. I also bought an original bonus disk. I use nintendont to be able to play in my EUR Wii. I also emulate the memcard with nintendont. When I introduce Celebi Disk, It opens the main screen and when I press the A button, it turns black screen. I tried all configs, with native on and off, 59, etc. 

Any tips?
